http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1022,6 +1041,13 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
         )
         .run(helmAuthContext)
 
+      // Currently we always retry.
+      // The main failure mode here is helm install, which does not have easily interpretable error codes
+      _ <- tracedRetryF(retryConfig)(
```

Comment:
Should we retry the setup chart and nginx helm installs too?

Reply:
I thought about it, but I think we've mainly seen it around the galaxy-specific helm install and I wanted to be conservative on this pass